.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: python
+sudo: required
+dist: trusty
+
+cache:
+  directories:
+    - /var/lib/docker/image
+
+services:
+  - docker
+env:
+  global:
+    - DOCKER_VERSION=1.11.1-0~trusty
+    - DOCKER_COMPOSE_VERSION=1.7.1
```

Comment:
Can we just use the default docker version instead of pinning to those specific versions?